test/runtests.jl

```diff
@@ -13,15 +13,29 @@ else
     typemax(Csize_t)
 end
 
+
+test_names = copy(tests)
+for (i, test) in enumerate(test_names)
+    if contains(test, "stdlib") && endswith(test, "runtests")
+        test_names[i] = split(test, "/")[end-2]
+    end
+end
```

Comment:
It's a bit funny to transorm test names to "runtest.jl" files (in "choosetest.jl"), and then back here to a name. Maybe infering the test file name could be implemented only here in this file?